### DISCUSSION TITLE
Tracking user by city name.

### DISCUSSION THREAD

**Discussion by ardhichoiruddin** [umami-software/umami#1533] [Ideas]
Hi.

I suggest to add user tracking with the name of the city. 
thank you.

**Comment by MagicLike** [comment]
I don't think this is compliant with Umami's privacy ideology.

**Comment by rohandebsarkar** [comment]
Main discussion: #1073.
Looks like this might get implemented in the near future: #1522.

**Reply by MagicLike** [reply]
Ok, than I was wrong. Sorry...

(idk., but are city names considered as personal information in the GDPR?)

**Reply by mikecao** [reply]
I think we should be able to do it since our data is anonymized.
